src/Controllers/LoggerController.js

```diff
@@ -4,6 +4,8 @@ import AdaptableController from './AdaptableController';
 import { LoggerAdapter } from '../Adapters/Logger/LoggerAdapter';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
+const logStringTruncateLength = 1000;
```

Comment:
i know, i know, it should be adjustable.  but this is the limit in parse.com and its a good start.  If someone needs to adjust they can do a pr??